### DISCUSSION TITLE
Mark issues that are not planned?

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-platform/eclipse.platform.swt#1308] [General]
I created https://github.com/eclipse-platform/eclipse.platform.swt/issues/1307 and I don't expect anything to happen with it. Should the issue have a particular label? Or something else done to it.

Should I have raised the whole thing as a discussion in the first place?

**Comment by laeubi** [comment]
If you like you can add "help wanted" or just see this as a reminder so if anyone is searching for this particular issue at laest can see it is a known problem.
